Signal/src/ViewControllers/ConversationView/Cells/OWSAudioMessageView.h

```diff
@@ -0,0 +1,26 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSAudioAttachmentPlayer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class ConversationViewItem;
+@class TSAttachmentStream;
+
+@interface OWSAudioMessageView : UIView
```

Comment:
This code was largely pulled unchanged from the old video adapter, which was also used to display audio attachments.